Project/backend/sandbox-server/sandbox-key.txt

```diff
@@ -0,0 +1 @@
+@p]Mylf#CqGi&jpSf_tm~l.U-2;S#>
```

Comment:
Can you remove these / add them back to the git ignore so the keys aren't in the repo?